subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CacheTaskArchiveErrorIntegrationTest.groovy

```diff
@@ -57,7 +59,7 @@ class CacheTaskArchiveErrorIntegrationTest extends AbstractIntegrationSpec {
         then:
         fails "customTask"
 
-        failureHasCause("Failed to store cache entry for task ':customTask'")
+        failureHasCause(matchesRegexp("Failed to store cache entry [0-9a-z]+ for task ':customTask': Could not pack tree 'output'.*"))
```

Comment:
That cache key regex looks weird to me. If we want to be strict, then it should be `[0-9a-f]+`. If we want to be lenient, `\w+` should suffice. If we want to go with the stricter variant, then extracting it to a constant might make sense, too.

Reply:
Used stricter version and moved it to a constant.